packages/mobile/src/screens/personal-info/components/hero/styles.ts

```diff
@@ -0,0 +1,42 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    height: 194,
+  },
+  flag: {
+    height: 139,
```

Comment:
Those look like magic numbers. It would be hard to maintain consistency across different devices. I'd rather go with relative numbers or refactor layout flex values